src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -703,6 +719,12 @@ void EstimatorChecks::checkGps(const Context &context, Report &reporter, const s
 					    events::ID("check_estimator_gps_multiple_spoofing_indicated"),
 					    events::Log::Critical, "GPS reports multiple spoofing indicated");
 
+		if (!_gps_spoofed) {
```

Comment:
I think we should be getting this event any time it transitions to spoofed right?